### PR TITLE
support to i3s resources

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Extended Support to below resources for I3s API Version 2000.
Deployment Plan
Validated All Use Cases. 

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,9 +3,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 # [v1.6.0](Unreleased)
 ### Notes
-- This release supports API2200 minimally where we can use OneView API2200 with this SDK. No new fields are added/deleted to support API2200.
+- This release supports API2200 minimally where we can use OneView API2200 with this SDK and ImageStreamer REST API version 2000 (I3S v5.40).  
+  This release adds variable.tf file to provide easy access for adding Appliance Credentials through environment variables or during runtime. 
+  No new fields are added/deleted to support API2200. 
 
 ### Oneview Features supported
+- Deployment Plans
```
Comment (yuvirani) [inline CHANGELOG.md:11]: Can you move this to "Image Streamer features supported" section?

File: endpoints-support.md [hunk 2]
```diff
@@ -139,10 +139,10 @@
 
 ## HPE Synergy Image Streamer
 
-| Endpoints                                                                       | Verb     | V200 | V300 | V500 |V600 |V800 |V1000 |V1020 |V1600 |
+| Endpoints                                                                       | Verb     | V200 | V300 | V500 |V600 |V800 |V1000 |V1020 |V1600 |V2000 |
```
Comment (yuvirani) [inline endpoints-support.md:142]: Add column for V2000, its messed up.

Comment (nabhajit-ray) [reply endpoints-support.md:142]: fix the endpoint file

Comment (AsisBagga) [reply endpoints-support.md:142]: done

File: CHANGELOG.md (hunk 1 above)
Comment (AsisBagga) [reply CHANGELOG.md]: done

**Review by nabhajit-ray — APPROVED** [review]

**Review by yuvirani — APPROVED** [review]

**Pull request merged**